OSVR-Unity/Assets/OSVRUnity/src/ClientKit.cs

```diff
@@ -30,6 +30,7 @@ public class ClientKit : MonoBehaviour
             public string AppID;
 
             private OSVR.ClientKit.ClientContext contextObject;
+            private static ClientKit _instance;
```

Comment:
OK, sounds good. I figured that they were null by default given the "safety" promises of the managed runtime, but I didn't have enough C# experience to be certain.